addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -73,6 +73,9 @@ function factory(dependencies) {
             if (this.threadView.channelInvitationForm.component) {
                 return;
             }
+            this.update({
+                isInvitePopoverOpen: !this.isInvitePopoverOpen,
+            });
```

Comment:
This was previously only used to open it, not as a toggle. The rest of the method is not applicable when closing it, please don't make it a toggle and create a separate close method.